run_chunked_eval.py

```diff
@@ -104,7 +99,6 @@ def main(model_name, strategy, task_name, eval_split):
         output_folder='results-normal-pooling',
         eval_splits=[eval_split],
         overwrite_results=True,
-        batch_size=BATCH_SIZE,
```

Comment:
I think this should not be remove as here is another batch size argument:
https://github.com/jina-ai/late-chunking/blob/8237bf2c36d46e10d93220d6ee904d7d53210f83/chunked_pooling/mteb_chunked_eval.py#L105
I know it is a bit messy ...

Reply:
that's annoying, but OK.